__tests__/app.test.js

```diff
@@ -316,4 +316,76 @@ describe('POST /api/articles/:article_id/comments', () => {
     });
     //username must be a string - optional test
 
+});
+
+describe('GET /api/articles (queries)', () => {
+    test('status 200, default sort by date and default sort order is descending ', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.articles).toBeSortedBy('created_at',{descending: true,})
+        });
+    });
+    test('status 200, sorted by and sort order is as per passed query ', () => {
+        return request(app)
+        .get('/api/articles?sort_by=author&order=asc')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.articles).toBeSortedBy('author')
+        });
+    });
+    test('status 200, filtered by topic passed in the query ', () => {
+        return request(app)
+        .get('/api/articles?topic=cats')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.articles).toEqual(
+                expect.arrayContaining([
+                    expect.objectContaining({
+                        author: expect.any(String),
+                        title: expect.any(String),
+                        article_id: expect.any(Number),
+                        topic: 'cats',
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        comment_count: expect.any(Number),
+                    })
+                 ])
+            );
+        });
+    });
+    test('status 400, sort_by and/or order doesnt exist', () => {
```

Comment:
stick an extra test in for an invalid order query as you've not currently tested for it